front/components/assistant_builder/AssistantBuilderPreviewDrawer.tsx

```diff
@@ -241,38 +232,36 @@ export default function AssistantBuilderRightPanel({
                     <Separator />
                   </>
                 )}
-                {template &&
-                  multiActionsMode &&
-                  template.presetActions.length > 0 && (
-                    <div className="flex flex-col gap-6">
-                      <Page.SectionHeader title="Add those tools" />
-                      {template.presetActions.map((presetAction, index) => (
-                        <div className="flex flex-col gap-2" key={index}>
-                          <div>{presetAction.help}</div>
-                          <TemplateAddActionButton
-                            action={presetAction}
-                            addAction={(presetAction) => {
-                              const action = getDefaultActionConfiguration(
-                                presetAction.type
-                              );
-                              if (!action) {
-                                // Unreachable
-                                return;
-                              }
-                              action.name = presetAction.name;
-                              action.description = presetAction.description;
-                              setAction({
-                                type: action.noConfigurationRequired
-                                  ? "insert"
-                                  : "pending",
-                                action,
-                              });
-                            }}
-                          />
-                        </div>
-                      ))}
-                    </div>
-                  )}
+                {template && template.presetActions.length > 0 && (
+                  <div className="flex flex-col gap-6">
+                    <Page.SectionHeader title="Add those tools" />
```

Comment:
`nit` sometimes it's tool with an upper `T` and sometimes not 🧐.

Reply:
Will fix 👍🏻